### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by invadersmustdie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# MQTT Blackbox Exporter
+# MQTT Blackbox Exporter [![Build Status](https://travis-ci.org/inovex/mqtt_blackbox_exporter.png?branch=master)](https://travis-ci.org/inovex/mqtt_blackbox_exporter)
+
```
Comment (arnisoph) [inline README.md]: Could you set all badges on the same row please?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+[![Go Report Card](https://goreportcard.com/badge/github.com/inovex/mqtt_blackbox_exporter)](https://goreportcard.com/badge/github.com/inovex/mqtt_blackbox_exporter)
+[![Docker Pulls](https://img.shields.io/docker/pulls/inovex/mqtt_blackbox_exporter.svg?maxAge=604800)](https://img.shields.io/docker/pulls/inovex/mqtt_blackbox_exporter.svg?maxAge=604800)
```
Comment (arnisoph) [inline README.md]: please change target URL to https://hub.docker.com/r/inovex/mqtt_blackbox_exporter/

File: README.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
-# MQTT Blackbox Exporter
+# MQTT Blackbox Exporter [![Build Status](https://travis-ci.org/inovex/mqtt_blackbox_exporter.png?branch=master)](https://travis-ci.org/inovex/mqtt_blackbox_exporter)
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/inovex/mqtt_blackbox_exporter)](https://goreportcard.com/badge/github.com/inovex/mqtt_blackbox_exporter)
```
Comment (arnisoph) [inline README.md]: please change the target URL to https://goreportcard.com/report/github.com/inovex/mqtt_blackbox_exporter

File: README.md [hunk 4]
```diff
@@ -41,32 +44,32 @@ docker run --rm -it -p 9214:9214 -v ${PWD}/:/data/ inovex/mqtt_blackbox_exporter
 ```
 $ curl -s http://127.0.0.1:9214/metrics
 ...
-# HELP probe_mqtt_completed Number of completed probes.
```
Comment (arnisoph) [inline README.md:44]: good catch! 👍

File: README.md (hunk 2 above)
Comment (invadersmustdie) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (invadersmustdie) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (invadersmustdie) [reply README.md]: fixed

**Pull request merged**